src/index.ts

```diff
@@ -1,129 +1,97 @@
-import { program } from 'commander';
-
-import config from '@config/config.json';
+import IRestartCondition from '@interfaces/restart-conditions/IRestartCondition';
+import IAlertService from '@interfaces/services/alert/IAlertService';
+import IGlobalNetworkService from '@interfaces/services/global-network/IGlobalNetworkService';
 import ILoggerService from '@interfaces/services/logger/ILoggerService';
+import IMetagraphService from '@interfaces/services/metagraph/IMetagraphService';
+import ISeedlistService from '@interfaces/services/seedlist/ISeedlistService';
 import ISshService from '@interfaces/services/ssh/ISshService';
-import CheckMetagraphHealth from '@jobs/check-metagraph-health/CheckMetagraphHealth';
-import { NoAlertsService } from '@services/alert/NoAlertsService';
-import ConstellationGlobalNetworkService from '@services/global-network/ConstellationGlobalNetworkService';
-import { ConsoleLoggerService } from '@services/logger/ConsoleLoggerService';
-import { FileLoggerService } from '@services/logger/FileLoggerService';
-import ConstellationMetagraphService from '@services/metagraph/ConstellationMetagraphService';
-import NoSeedlistService from '@services/seedlist/NoSeedlistService';
-import { Ssh2Service } from '@services/ssh/Ssh2Service';
-import { NetworkNames } from '@shared/constants';
-
-program
-  .version('1.0.0')
-  .option('-fr, --force_restart', 'Force complete metagraph restart', false)
-  .option('-d, --dev_mode', 'Start application in dev mode', false);
-
-program.parse(process.argv);
-const options = program.opts();
 
-const intializeSshConnections = async (
-  logger: ILoggerService,
-): Promise<ISshService[]> => {
-  const { nodes: metagraphNodes } = config.metagraph;
-  const sshServices: ISshService[] = [];
-  logger.info(
-    `##################### INITIALIZING SSH CONNECTIONS #####################`,
-  );
-
-  const promises = [];
-  const initializeSshConnection = async (idx: number) => {
-    logger.info(`Starting ssh connection to node ${idx + 1}`);
-    const sshService = new Ssh2Service(
-      idx + 1,
-      metagraphNodes[idx],
-      logger,
-      `/home/${metagraphNodes[idx].username}/code`,
+import CheckMetagraphHealth from './check-metagraph-health/CheckMetagraphHealth';
+import { MonitoringConfiguration, Configs } from './MonitoringConfiguration';
+
+export default class MonitoringApp {
+  public configuration: MonitoringConfiguration;
+  public forceRestart: boolean;
+
+  constructor(
+    configs: Configs,
```

Comment:
"config" is both singular and plural in english. 
```suggestion
    config: Config,
```

Reply:
Got it